translations/handleDesktopTranslations.sh

```diff
@@ -12,16 +12,58 @@ gpg --list-keys
 git clone git@github.com:nextcloud/desktop.git
 cd desktop
 
-lupdate-qt5 src/gui/ src/cmd/ src/common/ src/crashreporter/ src/csync/ src/libsync/ -ts translations/client_en.ts
+# Generate source translation files for master and stable branches
+mkdir /branches
+
+versions="stable-2.6 master"
+if [[ -f '.tx/backport' ]]; then
+  versions="$(cat .tx/backport) master"
+fi
+
+for version in $versions
+do
+  # skip if the branch doesn't exist
+  if git branch -r | egrep "^\W*origin/$version$" ; then
+    echo "Valid branch"
+  else
+    echo "Invalid branch"
+    continue
+  fi
+
+  git checkout $version
+
+  lupdate-qt5 src/gui/ src/cmd/ src/common/ src/crashreporter/ src/csync/ src/libsync/ -ts /branches/$version.ts
+done
+
+# Merge source translation files and filter duplicates
+lconvert-qt5 -i /branches/*.ts -o translations/client_en.ts
 
 # push sources
 tx push -s
 
-# pull translations
-tx pull -f -a --minimum-perc=25
+# undo local changes
+git checkout -f --
+
+# reverse version list to apply backports
+backportVersions=$(echo $versions | awk '{for(i=NF;i>=1;i--) printf "%s ", $i;print ""}')
```

Comment:
Why reverse it? Should not be needed?

Reply:
The version list was reversed in the server translations, but I'm not sure why :see_no_evil: 

https://github.com/nextcloud/docker-ci/blob/master/translations/handleTranslations.sh#L55

Reply:
As we push to the corresponding branches, I do not see any need of any order, could also be random, or?

Reply:
Nothing I can think of right now.

Reply:
Then, @misch7 please kill it, as it just confuses us :see_no_evil:

Reply:
I wanted to be consistent with the other scripts but yeah, then I'll just kill it 😜